load-fuels-and-expenses-from-repsol-xls/load-fuels-and-expenses-from-respol-xls.py

```
@@ -574,10 +583,105 @@ def load_product_to_fuel_types():
     return get_json_from_file(path)
 
 
-def load_locations():
-    path = os.path.join(os.path.dirname(__file__), "maps", "repsol-locations-bd.json")
-    return get_json_from_file(path)
+def load_locations(codes_list: list):
+    basic_user = os.environ.get("BASIC_AUTH_USER", "admin")
+    basic_password = os.environ.get("BASIC_AUTH_PASS", "admin")
+
+    headers = {
```

Ruff (0.7.0)

Local variable `headers` is assigned to but never used

Remove assignment to unused variable `headers`

(F841)